src/Elastic.Apm/Report/PayloadSenderV2.cs

```diff
@@ -55,6 +55,9 @@ public PayloadSenderV2(IApmLogger logger, IConfigurationReader configurationRead
 			servicePoint.ConnectionLimit = 20;
 
 			_httpClient = new HttpClient(handler ?? new HttpClientHandler()) { BaseAddress = serverUrlBase };
+			_httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue($"elasticapm-{Consts.AgentName}", _service.Agent.Version));
+			_httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("http-lib",
+				typeof(HttpClient).Assembly.GetName().Version.ToString()));
```

Comment:
On my machine (.NET Core Runtime 2.1.11)  
- `typeof(HttpClient).Assembly.GetName().Version.ToString()` returns `4.2.1.0`
- `typeof(HttpClient).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version` returns `4.6.27617.02`
- `typeof(HttpClient).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion` returns `4.6.27617.02 @BuiltBy: dlab14-DDVSOWINAGE051 @Branch: release/2.1-MSRC @SrcCode: https://github.com/dotnet/corefx/tree/c6cf790234e063b855fcdb50f3fb1b3cf
ac73275`

Which makes me wonder what would be more useful to see  `4.2.1.0` or `4.6.27617.02`? WDTY?

Reply:
So, I checked it with .Net Core 2.2 and 3.0 and got the following results:

```
.Net Core 3.0
4.2.1.0
4.700.19.30308
3.0.0-preview6.19303.8+d47cae744ddfb625db8e391cecb261e4c3d7bb1c

.Net Core 2.2
4.2.1.0
4.6.27618.01
4.6.27618.01 @BuiltBy: dlab14-DDVSOWINAGE054 @Branch: release/2.2-MSRC @SrcCode: https://github.com/dotnet/corefx/tree/9b5053f09fac4a2d615619197924072b9571da78
```

So, I agreed with you that assembly file version is more suitable for our use case. Will change it